MediaBrowser.Providers/TV/SeriesMetadataService.cs

```diff
@@ -137,6 +137,8 @@ private async Task FillInMissingSeasonsAsync(Series series, CancellationToken ca
             // Loop through the unique season numbers
             foreach (var episode in episodesInSeriesFolder)
             {
+                await episode.RefreshMetadata(new MetadataRefreshOptions(new DirectoryService(FileSystem)), cancellationToken).ConfigureAwait(false);
```

Comment:
This shouldn't be necessary. The missing seasons is supposed to execute after metadata has already been refreshed.

Even if this fixes it I'm not sure it's the correct solution.

Reply:
Then i will probe a little further to see if i can find out why the metadata is not already fetched.